app/bundles/CoreBundle/Translations/en_US/flashes.ini

```diff
@@ -6,6 +6,7 @@ mautic.core.error.locked="%name% is currently checked out by <a href='%contactUr
 mautic.core.error.not.valid="Form is not valid. Fill in all required fields."
 mautic.core.error.notfound="Item was not found."
 mautic.core.error.delete.error="Item could not be deleted. %error%"
+mautic.core.error.max.field="Too many fields !"
```

Comment:
This should be something along the lines of "You have reached the limit of custom fields allowed by your database."